samples/next-ab-testing/components/ButtonExperiment.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Suggesting we consider naming these files based on what feature they are showing off, vs. more general concepts like `Button` and `Color`

Reply:
Have a good name only for ButtonExperiment -> PlayAndWinHooksDemonstration
Didn't come up with the better names for TextExperiment & ColorExperiment

Reply:
`ButtonExperiment` => `PlayAndWinHooksUsage`
`ColorExperiment` => `BasicUsage`

What functionality does `TextExperiment` show off?

Reply:
It is BasicUsage as well